File: ServerSpace/src/main/java/Models/General/MonedaModel.java
package Models.General;

import com.fasterxml.jackson.annotation.JsonProperty;

import EntityLayer.MonedaEntity;

public class MonedaModel {
    public MonedaModel() {
        this.monedaId = 0;
        this.codigo = "";
        this.codigoInterno = "";
        this.simbolo = "";
        this.nombre = "";
    }

    public MonedaModel(MonedaEntity Ent) {
        this.monedaId = Ent.getMonedaId();
        this.codigo = Ent.getCodigo();
        this.codigoInterno = Ent.getCodigoInterno();
        this.simbolo = Ent.getSimbolo();
        this.nombre = Ent.getNombre();
    }

    @JsonProperty("MonedaId")
    private int monedaId = 0;

    public int getMonedaId() {
        return monedaId;
    }

    public void setMonedaId(int monedaId) {
        this.monedaId = monedaId;
    }

    @JsonProperty("Codigo")
    private String codigo = "";

    public String getCodigo() {
        return codigo;
    }

    public void setCodigo(String codigo) {
        this.codigo = codigo;
    }

    @JsonProperty("CodigoInterno")
    private String codigoInterno = "";

    public String getCodigoInterno() {
        return codigoInterno;
    }

    public void setCodigoInterno(String codigoInterno) {
        this.codigoInterno = codigoInterno;
    }


    @JsonProperty("Simbolo")
    private String simbolo = "";

    public String getSimbolo() {
        return simbolo;
    }

    public void setSimbolo(String simbolo) {
        this.simbolo = simbolo;
    }

    @JsonProperty("Nombre")
    private String nombre = "";

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }
}

File: SharedApi/Space/Space.Server/Model/General/MonedaModel.cs
using Space.EntityLayer;
using System.Text.Json.Serialization;

namespace Space.Server.Model.General
{
    public class MonedaModel
    {
        public MonedaModel()
        {
            MonedaId = 0;
            Codigo = String.Empty;
            CodigoInterno = String.Empty;
            Simbolo = String.Empty;
            Nombre = String.Empty;
        }

        public MonedaModel(MonedaEntity Ent)
        {
            this.MonedaId = Ent.MonedaId;
            this.Codigo = Ent.Codigo;
            this.CodigoInterno = Ent.CodigoInterno;
            this.Simbolo = Ent.Simbolo;
            this.Nombre = Ent.Nombre;
        }
        [JsonPropertyName("MonedaId")]
        public Int32 MonedaId { get; set; }
        [JsonPropertyName("Codigo")]
        public String Codigo { get; set; }
        [JsonPropertyName("CodigoInterno")]
        public String CodigoInterno { get; set; }
        [JsonPropertyName("Simbolo")]
        public String Simbolo { get; set; }
        [JsonPropertyName("Nombre")]
        public String Nombre { get; set; }
    }
}
